libraries/SoaringController/SoaringController.cpp

```diff
@@ -0,0 +1,415 @@
+#include "SoaringController.h"
+
+extern const AP_HAL::HAL& hal;
+
+
+// ArduSoar parameters
+const AP_Param::GroupInfo SoaringController::var_info[] = {
+    // @Param: SOAR_ACTIVE
+    // @DisplayName: Is the soaring mode active or not
+    // @Description: Toggles the soaring mode on and off
+    // @Units: boolean
+    // @Range: 0 1
+    // @User: Advanced
+    AP_GROUPINFO("ACTIVE", 0, SoaringController, soar_active, 0),
```

Comment:
Thanks. However, in doing so may corrupt your own EEPROM if you loaded that into hardware. Heads up, you may want to check the params on your unit. This would only effect you, not anyone else, since.